delta/plugins/elasticsearch/src/main/scala/ch/epfl/bluebrain/nexus/delta/plugins/elasticsearch/model/ElasticSearchViewRejection.scala

```diff
@@ -126,7 +126,7 @@ object ElasticSearchViewRejection {
     *   the view id
     */
   final case class DifferentElasticSearchViewType(
-      id: Iri,
+      id: String,
```

Comment:
To be able to pass an IdSegment as well, thought the segment might not be fully expanded 🤔